tests/v2/validation/snapshot/README.md

```diff
@@ -17,14 +17,17 @@ rancher:
   insecure: true/optional
   cleanup: false/optional
 snapshotInput:
-  upgradeKubernetesVersion: "v1.27.6+rke2r1" # If left blank, the default version in Rancher will be used.
+  upgradeKubernetesVersion: "" # If left blank, the default version in Rancher will be used.
   snapshotRestore: "all" # Options include none, kubernetesVersion, all. Option 'none' means that only the etcd will be restored.
   controlPlaneConcurrencyValue: "15%"
   workerConcurrencyValue: "20%"
   controlPlaneUnavailableValue: "1"
   workerUnavailableValue: "10%"
   recurringRestores: 1 # By default, this is set to 1 if this field is not included in the config.
-  replaceWorkerNode: false
+  replaceNodes:
```

Comment:
see nit in shepherd about naming of this struct

Reply:
Resolved